sdks/python/apache_beam/dataframe/io.py

```diff
@@ -521,7 +521,7 @@ def expand(self, pcoll):
     return pcoll | fileio.WriteToFiles(
         path=dir,
         file_naming=fileio.default_file_naming(name),
-        sink=_WriteToPandasFileSink(
+        sink=lambda _: _WriteToPandasFileSink(
```

Comment:
Based on the ease of making this mistake, is the `WriteToFiles` API a hazard to other contributors and users?

Reply:
Yeah I think so. I'm not sure how to address it though. Should we detect and raise when this mode is used with non global windows? I'll file a jira for this.

Reply:
BEAM-12072